src/operations/resize.jl

```diff
@@ -79,8 +79,8 @@ function padrange(range::AbstractUnitRange, pad)
 end
 
 function applyaffineview(op::Resize{N}, img::AbstractArray{T,N}, param) where {T,N}
-    Rin, Rout = CartesianRange(axes(img)), CartesianRange(op.size)
-    sf = map(/, (last(Rout)-first(Rout)+1).I, (last(Rin)-first(Rin)+1).I)
+    Rin, Rout = CartesianIndices(axes(img)), CartesianIndices(op.size)
+    sf = Tuple(last(Rout)) ./ Tuple(last(Rin) - first(Rin) + first(Rout))
```

Comment:
This doesn't quite look equivalent on first glance. is it?

Reply:
It turns out that `CartesianIndices(::Tuple)` returns an index array whose first element is `CartesianIndex(1, 1)`. But yes,  your're right, what we need here is to substitute the 1-base `CartesianIndex(1, 1)` and it has nothing to do with `Rout`.

Reply:
fixed in https://github.com/Evizero/Augmentor.jl/pull/31/commits/67e97846c1025492cad36196e28fa6d5945d1175